.scalafmt.conf

```diff
@@ -0,0 +1,11 @@
+style = defaultWithAlign
+//@formatter on
+align                      = more
+danglingParentheses        = true
+docstrings                 = JavaDoc
+indentOperator             = spray
+maxColumn                  = 120
```

Comment:
Hmmm this doesn't seem to be applied everywhere, f.e. `HostBalancer.scala` seem to be 80